components/dependency_injection/autowiring.rst

```diff
@@ -0,0 +1,399 @@
+.. index::
+    single: DependencyInjection; Autowiring
+
+Using the Autowiring
+====================
+
+Autowiring allows to register services in the container with minimal configuration.
+It is useful in the field of `Rapid Application Development`_, when designing prototypes
+in early stages of large projects. It makes it easy to bootstrap an app service
+graph and eases refactoring.
+
+Imagine you're building an API to publish statuses on a Twitter feed, obfuscated
+with `ROT13`.. (a special case of the Caesar cipher).
```

Comment:
I'd remove the part about `Caesar cipher` and I'd add a link in `ROT13` to https://en.wikipedia.org/wiki/ROT13